web/src/beta/features/Editor/tabs/publish/Nav/PublishModal/index.tsx

```diff
@@ -1,9 +1,9 @@
 import { useMemo } from "react";
 
 import Button from "@reearth/beta/components/Button";
+import ToggleButton from "@reearth/beta/components/Fields/Toggle";
```

Comment:
Fields should be lowercased since the directory is a collection of fields, not a Fields component 👍🏻

Reply:
done

Reply:
https://github.com/reearth/reearth/pull/627/commits/1e3d61c9ddccd2fb7f0e29ccb9a18fe4b1177389

Reply:
https://github.com/reearth/reearth/pull/627/commits/8f19fa776aff266a9d472bfea6db207b4e308984